src/components/controls.vue

```diff
@@ -8,8 +8,9 @@
         :key="index"
         :transfer-data="{type: control.type}"
         v-if="control.label.toLowerCase().includes(filterQuery.toLowerCase())"
+        :class="[ control.type ]"
```

Comment:
Was this added so the element can be selected in tests? If so, why not use the same `data-test` attribute as above?

Reply:
Binding a class to each nodes allows us to select a node from the tool panel. If we used `data-test` to select each node for tests, we would have to manually attach `data-test="nameOfName"` to each node.